HousingSearchApi.Tests/V1/E2ETests/Steps/GetAssetSteps.cs

```diff
@@ -74,9 +74,9 @@ public async Task WhenNoOfBedSpacesIsProvided(int numberOfBedSpaces)
             _lastResponse = await _httpClient.GetAsync(route).ConfigureAwait(false);
         }
 
-        public async Task WhenGroundFloorIsProvided(bool isGroundFloor)
+        public async Task WhenFloorNoIsProvided(int floorNo)
         {
-            var route = new Uri($"api/v1/search/assets/all?groundFloor={isGroundFloor}&pageSize={1}",
+            var route = new Uri($"api/v1/search/assets/all?groundFloor={floorNo}&pageSize={1}",
```

Comment:
Should the query parameter name be floorNumber?

Reply:
It looks like FloorNo was added by someone called Alex Demetriou. The asset shared package I took it from would also need to be updated which a lot of other apis depend on. Could just make the change in this repo but that would leave things a little disjointed.

Reply:
that's fair enough - we can make the name change afterwards. However, according to the GetAllAsetListRequest class, the query param should be called floorNo and not groundFloor as this URI suggests.

Reply:
Yes I see that. I've made the change.